llm-server/models/chatbot.py

```diff
@@ -0,0 +1,25 @@
+import uuid
```

Comment:
I think it would be more suitable to have the shared library here instead of workers. Because workers code will rarely change. What do you suggest @gharbat ?